### PR TITLE
How much do networks vary in space?

### PR DESCRIPTION
**Pull request by DominiqueCaron**
#4 Recreated a branch + PR to make the name consistent with other branches ("SECTION" instead of "Section")

### CODE REVIEW

File: manuscript.md [hunk 1]
```diff
@@ -52,6 +52,8 @@
 
 Networks can vary either in their structural properties (e.g. changes in connectance or degree distribution) or in their composition (e.g. changes in nodes and links identity). Interestingly, variation in the structural properties of ecological networks tend to respond mostly to changes in the size of the network. First, the number of links in ecological networks scales with the number of species [@MacDonald2020RevLin; @Brose2004UniSpa]. Second, connectance and size are driving the rest of the structural properties of ecological networks [@Poisot2014WheEco; @Dunne2002FooStr; @Riede2010ScaFoo]. So, network properties do vary in space, but this variation is mainly explained by the spatial variation in species richness.
```
Comment (gottacatchenall) [inline manuscript.md:53]: > but this variation is mainly explained by the spatial variation in species richness.

I think this probably needs a citation

File: manuscript.md [hunk 2]
```diff
@@ -52,6 +52,8 @@
...
 
+On a lower level, species turnover in space obviously results in changes in the composition of ecological networks. But, this is not the only reason networks composition varies [@Poisot2015BeySpe]. Intraspecific variations can result in interaction turnovers without changes in species composition [ADD REF]. Similarly, changes in species abundances can lead to variation in interactions [ADD REF].
```
Comment (gottacatchenall) [inline manuscript.md:55]: > Similarly, changes in species abundances can lead to variation in interactions [ADD REF].

I'm thinking [Vasquez et al 2007 in Oikos](http://www.dx.doi.org/10.1111/j.0030-1299.2007.15828.x)

File: manuscript.md (hunk 1 above)
Comment (gottacatchenall) [inline manuscript.md:53]: > but this variation is mainly explained by the spatial variation in species richness

I think this is the primary point here, could be emphasized at the beginning

File: manuscript.md (hunk 2 above)
Comment (gottacatchenall) [inline manuscript.md:55]: > Intraspecific variations can result in interaction turnovers without changes in species composition [ADD REF]. 

I'm thinking [Bolnick et al 2012](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3088364/pdf/nihms285002.pdf)

**Review by gottacatchenall — COMMENTED** [review]
:+1:

**Review by gottacatchenall — CHANGES_REQUESTED** [review]
added a few notes

**Review by TanyaS08 — COMMENTED** [review]
I really really like this section. It ties/brings together a lot of the points raised earlier in the ms and sets the stage for discussions around space. I wonder if we could/should briefly discuss how much networks vary over different spatial scales...

**Comment by DominiqueCaron** [comment]
> .

Interesting. Something like discussed in [Galiana et al. 2018](https://www-nature-com.proxy3.library.mcgill.ca/articles/s41559-018-0517-3)? I feel that it is somehow a different question, but still very relevant and related. If we are not discussing it in another section (maybe in the spatial scale of prediction?), and we still have a couple of words we can add, I can surely find a spot to add this! Maybe I'll tweak the question a bit.

**Comment by TanyaS08** [comment]
> > I really really like this section. It ties/brings together a lot of the points raised earlier in the ms and sets the stage for discussions around space. I wonder if we could/should briefly discuss how much networks vary over different spatial scales...
> 
> Interesting. Something like discussed in [Galiana et al. 2018](https://www-nature-com.proxy3.library.mcgill.ca/articles/s41559-018-0517-3)? I feel that it is somehow a different question, but still very relevant and related. If we are not discussing it in another section (maybe in the spatial scale of prediction?), and we still have a couple of words we can add, I can surely find a spot to add this! Maybe I'll tweak the question a bit.

Firstly ref = A+ in terms of direction of thinking. I think you're right that it might be better suited to a different section but maybe something we can flag/keep in mind when we start integrating and include where needed?

**Review by gottacatchenall — APPROVED** [review]

**Review by TanyaS08 — APPROVED** [review]

**Pull request merged**